tools/file_packager.py

```diff
@@ -635,11 +635,7 @@ def generate_js(data_target, data_files, metadata):
           Module['FS_createPreloadedFile'](this.name, null, byteArray, true, true, function() {
             Module['removeRunDependency']('fp ' + that.name);
           }, function() {
-            if (that.audio) {
-              Module['removeRunDependency']('fp ' + that.name); // workaround for chromium bug 124926 (still no audio with this, but at least we don't hang)
-            } else {
```

Comment:
Why remove this? Don't we need the workaround if the bug is WONTFIX?

Reply:
The WONTFIX was for broken BlobBuilder.    According to the bug:

```
Well, BlobBuilder appears to be deprecated (probably for quite a while now) and is undefined in current Linux Chrome dev M35. See comment #16 and http://updates.html5rocks.com/2012/06/Don-t-Build-Blobs-Construct-Them.

As attached, I modified the script in page.html to use the new Blob constructor instead, and the ogg file loads successfully and two sounds play, including when I replay.
```

And since we are using Blob constructor now there should no longer be an issue.

Reply:
I see. Yeah, that's probably right, then, though an earlier comment says

> it happens with the Blob constructor too, not just the (deprecated) BlobBuilder
> https://bugs.chromium.org/p/chromium/issues/detail?id=124926#c16

But the later comment says it works, and I do see it work locally when I try that modified page.html, so looks good.